voctocore/lib/videomix.py

```diff
@@ -45,9 +45,24 @@ def __init__(self):
 
         self.names = Config.getlist('mix', 'sources')
         self.log.info('Configuring Mixer for %u Sources', len(self.names))
+        
+        self.has_alpha_method = 0
+        
+        for idx, name in enumerate(self.names):
+            section = 'source.{}'.format(name)
+            self.has_alpha_method |= Config.has_option(section, 'alpha_method')
+
 
         pipeline = """
             compositor name=mix !
+        """
+        
+        if self.has_alpha_method:
+            pipeline += """
+                videoconvert !
```

Comment:
`videoconvert` will just be a NoOp if no conversion is needed. I think it would be fine to just add it to the pipeline independent of the alpha-configuration.